manifest/v1alpha/service/validation.go

```diff
@@ -12,6 +16,55 @@ func validate(s Service) *v1alpha.ObjectError {
 	return v1alpha.ValidateObject(validator, s, manifest.KindService)
 }
 
+// atLeastDailyContinuousFreq validates that RRULE frequency is DAILY or higher (WEEKLY, MONTHLY, YEARLY) and
+// that it has continuous occurrences.
+var atLeastDailyContinuousFreq = govy.NewRule(func(rule *rrule.RRule) error {
+	if rule == nil {
+		return nil
+	}
+
+	if rule.Options.Count > 0 || !rule.Options.Until.IsZero() {
+		return errors.New("rrule must have continuous occurrences")
```

Comment:
I don't get this condition (neither the returned error). Can you please explain?